game/analysis.py

```
@@ -50,13 +50,98 @@ def count_promotions(moves: list[str]) -> int:
+        }
+
+    move_analysis = []
+    mistakes = 0
+    blunders = 0
+    accuracy = 100
+    material_summary = []
+
+    for i in range(len(fen_history)):
+        material_summary.append(compute_material(fen_history[i]))
+
+    for i, move in enumerate(moves):
```

Ruff (0.15.18)

[warning] Loop control variable `move` not used within loop body

Rename unused `move` to `_move`

(B007)